### ISSUE TITLE
cant open mkv with gallery and file manager

### ISSUE THREAD

**Issue by qwertyuser1983** [SimpleMobileTools/General-Discussion#38]
Both app did not correctly add mkv to video (mimetype?), and I have VLC installed. The apps said no app for opening this type of file is installed. :(

Most other format played properly. :)

**Comment by tibbi** [comment]
Hah, mkv wasnt even at the table of mimetypes, Ive added it as video/x-matroska so it will be fixed when the apps are updated the next time. The sample mkv video I downloaded didnt play to me tho, does it work to you? If you open some other file and swipe to the video.

**Comment by qwertyuser1983** [comment]
@tibbi 

I am sorry, I may misinterpreted Your reply. I did not catch what You mean with "open some other files and swipe to the videos".

In my current case, I can play mkv files from stock file manager or XPlore, but not from Simple File Manager nor Simple Gallery.

I tried to share the mkv, but the same error persist.

**Comment by tibbi** [comment]
Imagine that you have some photo and a mkv video in the same folder. If you open the photo in Simple Gallery and swipe to the video, does the video play well?

**Comment by qwertyuser1983** [comment]
@tibbi yes, in Simple Gallery, confirmed that the mkv can be played by swiping right from the image file in the same folder.

Hmm, quite a peculiar bug here. 

:smile:
